scripts/tests/test_validate_docstrings.py

```diff
@@ -502,6 +502,20 @@ def no_punctuation(self):
         return "Hello world!"
 
 
+class BadExamples(object):
+
+    def npPd_import(self):
+        """
+        Provide example with numpy and pandas import
+
+        Examples
+        --------
+        import numpy as np
+        import pandas as pd
```

Comment:
The code in examples start by `>>>`

It'd be good to have tests for `import numpy` and `import pandas` (without the aliases), and have each of them separate. And also a case when something else is imported (e.g. `import datetime`)

Not sure if in the good docstrings in the tests we have examples somewhere. We should, to make sure the error is not reported when `import numpy`... is not present.